lighthouse-core/config/config-helpers.js

```diff
@@ -183,13 +183,14 @@ function resolveModule(moduleIdentifier, configDir, category) {
     return require.resolve(cwdPath);
   } catch (e) {}
 
-  const errorString =
-    'Unable to locate ' +
-    (category ? `${category}: ` : '') +
-    `${moduleIdentifier} (tried to require() from '${__dirname}' and load from '${cwdPath}'`;
+  let errorString = 'Unable to locate ' + (category ? `${category}: ` : '') +
+    `\`${moduleIdentifier}\`.
+     Tried to require() from these locations:
+       ${__dirname}
+       ${cwdPath}`;
```

Comment:
we might also spit out `NODE_PATH`, if it is set. L175 tries all of that.

Reply:
Sounds good. I started this but i'm going to split into a separate PR because NODE_PATH from the node side starts to get pretty complex.